Formula/arduino-cli.rb

```diff
@@ -17,10 +17,12 @@ class ArduinoCli < Formula
 
   def install
     commit = Utils.safe_popen_read("git", "rev-parse", "HEAD").chomp
-    system "go", "build", "-ldflags",
-           "-s -w -X github.com/arduino/arduino-cli/version.versionString=#{version} " \
-           "-X github.com/arduino/arduino-cli/version.commit=#{commit}",
-           "-o", bin/"arduino-cli"
+    ldflags = %W[
+      -s -w
+      -X github.com/arduino/arduino-cli/version.versionString=#{version}
+      -X github.com/arduino/arduino-cli/version.commit=#{commit}
+    ]
+    system "go", "build", *std_go_args, "-ldflags", ldflags
```

Comment:
Hello @SConaway, I've got some problems with this PR: now when I run `arduino-cli version` it shows 
```
arduino-cli Version: 0.0.0-git Commit: 
```
I think that the problem is this refactoring you made..

Reply:
I wasn't the one who refactored this, `dawidd6` did. I don't know much about `go`, but I'll play around and investigate some. Just checked and i get that too on my machine